gcp/appengine/frontend3/src/templates/vulnerability.html

```diff
@@ -230,17 +230,51 @@ <h3 class="mdc-layout-grid__cell--span-3">
 </turbo-stream>
 
 <script>
-  /*
-  We use `spicy-section` to make the packages content collapsible for mobile view,
-  but the issue is it collapse the content by default but we want to expaneded after
-  the page is loaded. So we decide to programmatically click on header of collapsed
-  packages after the page is loaded, and the content will be visible.
-  */
   document.addEventListener('turbo:load', function() {
+    /*
+    We use `spicy-section` to make the packages content collapsible for mobile view,
+    but the issue is it collapse the content by default but we want to expaneded after
+    the page is loaded. So we decide to programmatically click on header of collapsed
+    packages after the page is loaded, and the content will be visible.
+    */
     const package_headers = document.querySelectorAll('.package-header[affordance="collapse"][aria-expanded="false"]')
     package_headers.forEach((header) => {
       header.click()
     });
-  })
+
+    /*
+    Collapse fields if they have too many items. You can enable this feature by adding
+    an 'expandible-list' class to lists with long value.
+    */
+    const EXPANDIBLE_LIST_DEFAULT_LENGTH = 5
```

Comment:
Currently it shows at most 5 items by default, but we can change this number later.

Reply:
Rather than hard-coding a number, is it possible to take the size of the window into consideration? I think it's important for someone viewing a vulnerability record to be able to visually reason about all of the affected packages in a single page (where the window is reasonably large)

Reply:
This was more about expanding the affected packages, than collapsing the references (which I definitely think should be collapsed by default if more than say 5)